data/multus/001-multus.yaml

```diff
@@ -0,0 +1,181 @@
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: kube-multus-ds-amd64
```

Comment:
We currently use `multus` as the name. I would prefer keeping it since it looks so much nicer, but if you decide to just copy what is available from U/S, please add a code to the Multus component to delete old `multus` daemon sets.